functions/snippets/concepts/src/main/java/com/example/functions/concepts/eventpojos/GcsEvent.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.functions.concepts.eventpojos;
+
+import java.util.Date;
+
+// [START functions_helloworld_gcs_event]
```

Comment:
We're going to have multiple region tag source in DRIFT if we have multiple files that have the same tag. Do we need it in multiple places?

Reply:
Commented below.